jenkinsci/jenkins
# Update OpenSearch to use the newer Jenkins logo

**Tiny** PR to update the OpenSearch logo, which currently uses the old Hudson logo. This PR removes the icon field so the browser will default to the normal site favicon, although happy to set the icon field explicitly if preferred.

**Before**
<img width="95" alt="image" src="https://user-images.githubusercontent.com/43062514/169901119-3520c322-413d-4239-9742-6f9ed7b06917.png">

**After**
<img width="97" alt="image" src="https://user-images.githubusercontent.com/43062514/169901199-ab954407-9e4c-44b3-be50-76d9604054ef.png">

**How to reproduce**
- Open Jenkins in a browser that supports OpenSearch, e.g. Firefox
- Notice the Search field has a **+** button on it, add Jenkins
- Go to Settings > Search, see the old Hudson logo

### Proposed changelog entries

* Update OpenSearch to use the newer Jenkins logo



### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 



### Desired reviewers

@jenkinsci/sig-ux 



### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
